ui/app/src/components/shared/CostsTag.tsx

```diff
@@ -27,6 +27,14 @@ export const CostsTag: React.FunctionComponent<CostsTagProps> = (props: CostsTag
           maximumFractionDigits: 2
         }).format(resourceCosts?.costs[0].cost);
         costBadge = <Stack.Item style={{maxHeight: 18}} className="tre-badge">{formattedCost}</Stack.Item>
+      }else{
```

Comment:
nit: any reason why no spaces around `else`?

i.e. `}else{` vs `} else {`

(elsewhere we have the spaces)

Reply:
done. Got used to linter/auto formatting.